cyclonedx/model/__init__.py

```diff
@@ -594,25 +594,25 @@ class License:
         See the CycloneDX Schema definition: https://cyclonedx.org/docs/1.4/xml/#type_licenseType
     """
 
-    def __init__(self, *, id_: Optional[str] = None, name: Optional[str] = None,
```

Comment:
At this point, it would be better to have the variable (not the property) called `id_` 
because here in the scope, it would shadow pythons function `id()`.
As long as we don't call `id()` in this method scope, it is safe.

Reply:
As above.